docs/fundamentals/toc.yml

```diff
@@ -0,0 +1,1323 @@
+        - name: "Walkthrough: Creating a Dataflow Pipeline"
+          href: ../standard/parallel-programming/walkthrough-creating-a-dataflow-pipeline.md
+        - name: "How to: Unlink Dataflow Blocks"
+          href: ../standard/parallel-programming/how-to-unlink-dataflow-blocks.md
+        - name: "Walkthrough: Using Dataflow in a Windows Forms Application"
+          href: ../standard/parallel-programming/walkthrough-using-dataflow-in-a-windows-forms-application.md
+        - name: "How to: Cancel a Dataflow Block"
+          href: ../standard/parallel-programming/how-to-cancel-a-dataflow-block.md
+        - name: "Walkthrough: Creating a Custom Dataflow Block Type"
+          href: ../standard/parallel-programming/walkthrough-creating-a-custom-dataflow-block-type.md
+        - name: "How to: Use JoinBlock to Read Data From Multiple Sources"
+          href: ../standard/parallel-programming/how-to-use-joinblock-to-read-data-from-multiple-sources.md
+        - name: "How to: Specify the Degree of Parallelism in a Dataflow Block"
+          href: ../standard/parallel-programming/how-to-specify-the-degree-of-parallelism-in-a-dataflow-block.md
+        - name: "How to: Specify a Task Scheduler in a Dataflow Block"
+          href: ../standard/parallel-programming/how-to-specify-a-task-scheduler-in-a-dataflow-block.md
+        - name: "Walkthrough: Using BatchBlock and BatchedJoinBlock to Improve Efficiency"
+          href: ../standard/parallel-programming/walkthrough-using-batchblock-and-batchedjoinblock-to-improve-efficiency.md
+      - name: Using TPL with Other Asynchronous Patterns
+        href: ../standard/parallel-programming/using-tpl-with-other-asynchronous-patterns.md
+        items:
+        - name: TPL and Traditional .NET Framework Asynchronous Programming
+          href: ../standard/parallel-programming/tpl-and-traditional-async-programming.md
+        - name: "How to: Wrap EAP Patterns in a Task"
+          href: ../standard/parallel-programming/how-to-wrap-eap-patterns-in-a-task.md
+      - name: Potential Pitfalls in Data and Task Parallelism
+        href: ../standard/parallel-programming/potential-pitfalls-in-data-and-task-parallelism.md
+    - name: Parallel LINQ (PLINQ)
+      items:
+      - name: Introduction to PLINQ
+        href: ../standard/parallel-programming/introduction-to-plinq.md
+      - name: Understanding Speedup in PLINQ
+        href: ../standard/parallel-programming/understanding-speedup-in-plinq.md
+      - name: Order Preservation in PLINQ
+        href: ../standard/parallel-programming/order-preservation-in-plinq.md
+      - name: Merge Options in PLINQ
+        href: ../standard/parallel-programming/merge-options-in-plinq.md
+      - name: Potential Pitfalls with PLINQ
+        href: ../standard/parallel-programming/potential-pitfalls-with-plinq.md
+      - name: "How to: Create and Execute a Simple PLINQ Query"
```

Comment:
Now is probably a good time to correct the character casing of all these TOC entries, shouldn't they be sentence case?

Reply:
That can be done anytime, there's enough going on in this PR and that's super low priority, so I'd rather leave that for dedicated PRs after this one.  

@Youssef1313 I see you put 👍 on this comment, would you like to submit PR(s) to fix these titles after this PR is merged?

Reply:
@tdykstra Sure.